pkg/bundle/bundle_client.go

```diff
@@ -0,0 +1,63 @@
+package bundle
```

Comment:
I'd suggest renaming the file to `client.go`, so as to avoid stuttering.
> A package’s name provides context for its contents. For instance, the [bytes package](https://go.dev/pkg/bytes/) from the standard library exports the Buffer type. On its own, the name Buffer isn’t very descriptive, but when combined with its package name its meaning becomes clear: bytes.Buffer. If the package had a less descriptive name, like util, the buffer would likely acquire the longer and clumsier name util.BytesBuffer.

-- https://go.dev/blog/organizing-go-code

Reply:
Yes, good idea.